modules/@apostrophecms/schema/ui/apos/components/AposInputJoin.vue

```diff
@@ -69,6 +85,12 @@ export default {
         return { message: 'required' };
       }
 
+      // if the original status was disabled, no validation should change that
+      if (this.originalDisabled) {
```

Comment:
is this really an issue for all fields?

Reply:
I don't think so .. joins can be disabled by reaching their `max` limit, but then can be re-enabled by removing an item from the slat list. This two-ways-to-affect-state is unique to things that use the slat list.

A normal input is just going to be disabled from the get go and won't get re validated